doc/ref/configuration/master.rst

```diff
@@ -3994,9 +4012,11 @@ https://github.com/ytoolshed/range/wiki/%22yamlfile%22-module-file-spec
 
 .. code-block:: yaml
 
-  range_server: range:80
+    range_server: range:80
```

Comment:
This was a tab... caught by PyCharm. The same thing that fixed this, broke the one above...